commands/completion-scripts/completion.bash.gotmpl

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+: {{.App.Name}}
+
+# Macs have bash3 for which the bash-completion package doesn't include
+# _init_completion. This is a minimal version of that function.
+_{{.App.Name}}_init_completion() {
+  COMPREPLY=()
+  _get_comp_words_by_ref "$@" cur prev words cword
+}
+
+_{{.App.Name}}() {
+  if [[ "${COMP_WORDS[0]}" != "source" ]]; then
+    local cur opts base words
+    COMPREPLY=()
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    if declare -F _init_completion >/dev/null 2>&1; then
+      _init_completion -n "=:" || return
+    else
+      _{{.App.Name}}_init_completion -n "=:" || return
```

Comment:
Is there any possibility that .App.Name would have a space in it?  If so, quote this string.

Reply:
As-per https://github.com/StackExchange/dnscontrol/pull/2449#discussion_r1350392872, in `dnscontrol`'s case as-is, it won't. I think it's probably worth quoting though in this case, in case the hard-coding of `app.Name` is removed. will do.